### DISCUSSION TITLE
part-ii/2019-0/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#184] [Question comments]
Part II 2019 Algebraic Geometry: Paper 4, Section II, F 

https://questions.tripos.org/part-ii/2019-0/

**Comment by silkca** [comment]
The first half is entirely bookwork.

The second half is not very difficult - $G(p)$ has pretty much exactly the properties required for the expression of $\ell(np)$ to be true, although there is some care needed to ensure that the inequalities are in fact the right way around, and the final few parts directly use results from lectures which make them trivial.
